tidb-cloud/export-data-from-tidb-cloud.md

```diff
@@ -48,8 +48,15 @@ You can use the tool [Dumpling](/dumpling-overview.md) for data export.
     <div label="Serverless Tier">
 
     ```shell
-    tiup dumpling:v6.5.0 -h ${tidb_endpoint} -P 4000 -u ${user} -p ${password} --ca=${ca_path} -F 67108864MiB -t 4 -o ${export_dir} --filetype sql
+    tiup dumpling -h ${tidb_endpoint} -P 4000 -u ${user} -p ${password} -F 67108864MiB -t 4 -o ${export_dir} --filetype sql --consistency none
     ```
+   
+   > **Note:**
+   > 
+   > For exporting serverless tier data, you must ensure that the dumpling version is at least v6.5.0.
+   >
+   > When using dumpling:v6.5.0, you need to set `--ca=${ca_path}` in the command. To find the CA root path on your system, see [TLS Connections to Serverless Tier](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md#root-certificate-default-path).
```

Comment:
What about other Dumpling versions? For example v6.6.0, do users need to set `--ca=${ca_path}` in the command?

Reply:
@qiancai No, they needn't. Only v6.5.0 needs to set the CA path.